cdhweb/pages/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class PagesConfig(AppConfig):
+    name = 'cdhweb.pages'
+    # NOTE we have to relabel this to avoid a conflict with mezzanine's pages;
```

Comment:
thanks for this note; should we also note that template directory must match this? (in case we ever want to change it later)